web/api/report_server.thrift

```diff
@@ -574,6 +582,10 @@ service codeCheckerDBAccess {
                                7: optional bool  getDetails)
                                throws (1: codechecker_api_shared.RequestFailed requestError),
 
+  // Get report annotation values belonging to the given key.
+  // The "key" parameter is optional. If not given then the list of keys returns.
```

Comment:
Why wouldn't you just create a new function for querying the list of keys? Seems like a poor choice to merge these two functionalities into a single function, not to mention that the name of the function implies that the whole annotation map is going to be returned.

Reply:
I'll remove this whole function and `TestsuiteFilter.vue` completely for now, because these will be needed for the filtering option which we skip for now.